pdc/apps/changeset/filters.py

```diff
@@ -23,7 +23,7 @@ class ChangesetFilterSet(django_filters.FilterSet):
                                                 widget=widgets.DateTimeInput)
     changed_until = django_filters.MethodFilter(action='filter_committed_until',
                                                 widget=widgets.DateTimeInput)
-    comment = django_filters.CharFilter(name="comment", lookup_type="contains")
+    comment = django_filters.CharFilter(name="comment", lookup_expr="contains")
 
```

Comment:
Why we need change the lookup_type to lookup_expr, is this related to upgrade django? 
Is that because this lookup_type not work any more?